docs/docs/aws.md

```diff
@@ -669,6 +669,13 @@ Users can use catalog properties to override the defaults. For example, to confi
 --conf spark.sql.catalog.my_catalog.http-client.apache.max-connections=5
 ```
 
+**Note that for workloads with exceptionally high throughput against tables that S3 where you will likely to increase Retries, you will also like to increase the number of connections for the HTTP client**
+
+```shell
+spark.sql.catalog.my_catalog.http-client.apache.max-connections=200
```

Comment:
This doesn't look like an Iceberg setting from what I can tell.  If this is EMR specific, it should not be included here.

Reply:
It is a thing of AWS SDK and Spark ( not specifically to EMR). If you use Spark on your laptop writing to S3 and you are on this high throughput write scenario you will likely like to tune the parameter. 

I mean, we are speaking about a AWS docs in this page ( the parameter is quite specific to the S3 client of the AWS SDK).